app/models/find_api/site.rb

```diff
@@ -2,5 +2,15 @@ module FindAPI
   class Site < FindAPI::Resource
     belongs_to :recruitment_cycle, through: :provider, param: :recruitment_cycle_year
     belongs_to :provider, param: :provider_code
+
+    def name
+      location_name
+    end
+
+    def full_address
+      [address1, address2, address3, address4, postcode]
+        .reject(&:blank?)
+        .join(', ')
+    end
```

Comment:
We're moving away from using the V3 API where possible. Please can we use the public API for this? So TeacherTrainingPublicAPI::Location